MewsSystems/mews-flutter
# feat: Update radio component to use shared colors

#### Summary
### Updated radio component
- now uses shared colors
- added animation

<details><summary>Video</summary>

https://github.com/MewsSystems/mews-flutter/assets/9210422/819c5e20-0d1b-43ea-9cce-d039be6a2480


</details>


#### Testing steps

1. Check the Storybook story
2. Check the dark mode
3. Check disabled/enabled


#### Follow-up issues

None

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
